### DISCUSSION TITLE
Fedora doesn't have NeoVim 0.10 yet, is there a 0.9.5 supporting ver of NVChad?

### DISCUSSION THREAD

**Discussion by RaptaG** [NvChad/NvChad#2847] [Q&A] [answered]
Pretty straightforward

**Comment by siduck** [comment, marked as answer]
@RaptaG  the statusline's lsp message component uses 0.10, for 0.9.5 one it'll  not error! as for blankline plugin, it supports 0.10 only now, ig you can just use its  old commit.
